src/sinks/gcp/chronicle_unstructured.rs

```diff
@@ -132,6 +134,13 @@ pub struct ChronicleUnstructuredConfig {
     #[configurable(metadata(docs::examples = "production"))]
     pub namespace: Option<String>,
 
+    /// A set of labels that are attached to each batch of events.
+    ///
+    /// Both keys and values cannot be used in a template.
```

Comment:
```suggestion
```

We don't typically call that out in the description but instead rely on the parameter type to indicate it (`literal` vs `template`).

Reply:
Done.